ci/scripts/backwards-compat-test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+rc () {
+    psql -h localhost -p 4566 -d dev -U root -c "$@"
+}
+
+# Checkout <old-version>
+git checkout origin/v0.18.0-rc
```

Comment:
wonder how many versions back we should test.

Reply:
just 1 I suppose, the backward compatibility in theory is transitive among multiple versions by just always testing 2 consecutive version?

Reply:
If two versions are `v1.A.B` and `v1.C.D`, they must be backward compatible.
But if they are `v1.A.B` and `v2.C.D`, they can choose not to be backward compatible.